tests/components/bayesian/test_binary_sensor.py

```diff
@@ -203,7 +263,6 @@ async def test_sensor_numeric_state(
 
 async def test_sensor_state(hass: HomeAssistant) -> None:
     """Test sensor on state platform observations."""
-    prior = 0.2
```

Comment:
Instead of the magic numbers, pass `prior` to _test_sensor_state

Reply:
done